streaming/src/main/java/org/apache/carbondata/streaming/CarbonStreamRecordWriter.java

```diff
@@ -139,7 +139,9 @@ private void initialize(TaskAttemptContext job) throws IOException {
 
     segmentDir = CarbonTablePath.getSegmentPath(
         carbonTable.getAbsoluteTableIdentifier().getTablePath(), segmentId);
-    fileName = CarbonTablePath.getCarbonDataFileName(0, taskNo + "", 0, 0, "0", segmentId);
+    fileName = CarbonTablePath.getCarbonDataFileName(
+        0, taskNo + "", 0, 0, "0",
+        segmentId, CompressorFactory.NativeSupportedCompressor.SNAPPY.getName());
```

Comment:
revert this or use CCC default compressor

Reply:
fixed